indico/modules/rb/blueprint.py

```diff
@@ -75,6 +75,8 @@
 _bp.add_url_rule('/api/bookings/mine', 'my_bookings', bookings.RHMyUpcomingBookings)
 _bp.add_url_rule('/api/bookings/<int:booking_id>/<date>/<any(reject,cancel):action>',
                  'booking_occurrence_state_actions', bookings.RHBookingOccurrenceStateActions, methods=('POST',))
+_bp.add_url_rule('/api/bookings/<int:booking_id>/<date>/link/<int:event_id>',
+                 'booking_occurrence_link_actions', bookings.RHBookingOccurrenceLinkActions, methods=('POST',))
```

Comment:
Remove the "Actions" from the names here - "StateActions" above (which I assume you based your own naming on) is used because it covers multiple (reject/cancel) actions

Reply:
Removed 👍